variables.tf

```diff
@@ -34,6 +34,20 @@ variable "versioning_enabled" {
   description = "A state of versioning. Versioning is a means of keeping multiple variants of an object in the same bucket"
 }
 
+variable "logging" {
+  type = object({
+    enabled     = bool
```

Comment:
can you drop this? you can reach the same functionality with `null` as default value (as example see `cors_rule_inputs` in this module)

Reply:
Yeah, missed that detail. Thanks.